ui/admin/setup-edit.php

```diff
@@ -273,13 +273,13 @@
                     <span>&nbsp;</span>
                 </th>
                 <th scope="col" class="manage-column field-label">
-                    <span>Label<?php pods_help( __( "<h6>Label</h6>The label is the descriptive name to identify the Pod field.", 'pods' ) ); ?></span>
+                    <span><?php _e( 'Label', 'pods' ); pods_help( "<h6>" . __( 'Label', 'pods' ) . "</h6>" . __( 'The label is the descriptive name to identify the Pod field.', 'pods' ) ); ?></span>
                 </th>
                 <th scope="col" class="manage-column field-machine-name">
-                    <span>Name<?php pods_help( __( "<h6>Name</h6>The name attribute is what is used to identify and access the Pod field programatically.", 'pods' ) ); ?></span>
+                    <span><?php _e( 'Name', 'pods' ); pods_help( "<h6>" . __( 'Name', 'pods' ) . "</h6>" . __( 'The name attribute is what is used to identify and access the Pod field programatically.', 'pods' ) ); ?></span>
```

Comment:
Please use `esc_html_e()` instead of `_e()` for echoing values that do not require HTML.